.environment-scripts/install_esmf.sh

```diff
@@ -16,16 +16,15 @@ export ESMF_INSTALL_LIBDIR=$ESMF_INSTALL_PREFIX/lib
 export ESMF_INSTALL_BINDIR=$ESMF_INSTALL_PREFIX/bin
 export ESMF_NETCDF_INCLUDE=$NETCDF_INCLUDE
 export ESMF_NETCDF_LIBS="-lnetcdf -lnetcdff"
-export ESMF_BOPT=O3
+export ESMF_BOPT=O
```

Comment:
This is the updated way to specify this level of optimization when installing ESMF (see the documentation [here](https://earthsystemmodeling.org/docs/release/ESMF_5_2_0p1/ESMF_usrdoc/node9.html#SECTION00094000000000000000)).

Reply:
Though maybe I should set `ESMF_OPTLEVEL=3`.

Reply:
Probably a good idea to (hopefully) maintain existing behavior.

Reply:
Done.